src/main/resources/static/css/wisvch-dashboard.css

```diff
@@ -153,6 +153,12 @@ textarea:focus {
     transition: box-shadow 0.15s linear, -webkit-box-shadow 0.15s linear
 }
 
+#checkoutShoppingBasket {
+    margin-left: auto;
+    margin-right: 0;
+    float: right;
+}
+
```

Comment:
Why is this here?

Reply:
to align the button on the right, since the addition of the checkbox moved it more left.
![image](https://github.com/WISVCH/events/assets/58064813/0657e58c-ba9c-45a1-a4fd-d5dc884b1257), however i noticed that only the float is neccesary, so I will remove the margins.